src/kirby/components/floating-action-button/floating-action-button.component.tns.html

```diff
@@ -1,7 +1,5 @@
-<GridLayout class="fab-wrapper {{ isFloating ? '' : 'not-floating' }}">
-    <StackLayout class="fab-shadow" (loaded)="onViewLoaded($event)">
-        <StackLayout class="fab" [ngClass]="{ 'disabled' : disabled }">
-            <ng-content></ng-content>
-        </StackLayout>
+<StackLayout class="fab-shadow {{ isFloating ? '' : 'not-floating' }}" [kirbyElevation]="getElevation()">
```

Comment:
Be carefull with method calls vs properties as change detection might have performance impact:
```suggestion
<StackLayout class="fab-shadow {{ isFloating ? '' : 'not-floating' }}" [kirbyElevation]="elevation">
```